### ISSUE TITLE
Dockerfile is not working as-is

### ISSUE THREAD

**Issue by charles-spurgeon** [erichelgeson/mac-os9-docker#1]
Script references out-of-date ubuntu and QEMU versions so it can't build.

Updating those requires adding `pip` and `slirp-dev` to packages.

**Comment by erichelgeson** [comment]
Yes, I made this 4 years ago as a joke :) - If you want to PR any fixes I'll merge em!
